torch/export/exported_program.py

```diff
@@ -359,11 +359,11 @@ def _is_joint_ir_decomp(ep, joint_loss_index):
                         real_script_obj = ep.constants[node.meta["val"].name]
                     else:
                         real_script_obj = node.meta["val"].fake_val.real_obj
-                    fake_args.append(maybe_to_fake_obj(fake_mode, real_script_obj))
+                    retracing_args.append(real_script_obj)
```

Comment:
Why we no longer fakify script objects anymore in re-tracing?

Reply:
Because fakify_objects takes in non-fake TOrchbind objects and then fakify them.